core/src/main/java/ai/timefold/solver/core/impl/score/director/AbstractScoreDirector.java

```diff
@@ -313,6 +321,30 @@ protected void setWorkingEntityListDirty() {
         workingEntityListRevision++;
     }
 
+    @Override
+    public <Entity_, Value_> boolean isValueInRange(GenuineVariableMetaModel<Solution_, Entity_, Value_> variableMetaModel,
+            @Nullable Entity_ entity, @Nullable Value_ value) {
+        var valueRangeDescriptor = extractValueRangeDescriptor(variableMetaModel);
+        if (entity == null && valueRangeDescriptor.isEntityIndependent()) {
```

Comment:
Should it be `!valueRangeDescriptor.isEntityIndependent()` instead?

Reply:
If that’s the case, I wonder why it isn’t failing.

Reply:
I checked. No code actually passes `null` there.